static/js/factories/channels.js

```diff
@@ -25,24 +25,25 @@ export const makeChannel = (privateChannel: boolean = false): Channel => {
   const hasAvatar = casual.coin_flip
   return {
     // $FlowFixMe: Flow thinks incr.next().value may be undefined, but it won't ever be
-    name:                  `channel_${incr.next().value}`,
-    title:                 casual.title,
-    channel_type:          privateChannel ? "private" : "public",
-    description:           casual.description,
-    public_description:    casual.description,
-    num_users:             casual.integer(0, 500),
-    user_is_contributor:   casual.coin_flip,
-    user_is_subscriber:    casual.coin_flip,
-    user_is_moderator:     casual.coin_flip,
-    membership_is_managed: casual.coin_flip,
-    allow_widgets_ui:      casual.coin_flip,
-    ga_tracking_id:        casual.random_element(["UA-FAKE-01", null]),
-    avatar:                hasAvatar ? "http://avatar.url" : null,
-    avatar_small:          hasAvatar ? "http://avatar.small.url" : null,
-    avatar_medium:         hasAvatar ? "http://avatar.medium.url" : null,
-    banner:                casual.coin_flip ? "http://banner.url" : null,
-    widget_list_id:        casual.integer(4, 99),
-    allowed_post_types:    [LINK_TYPE_TEXT, LINK_TYPE_LINK, LINK_TYPE_ARTICLE]
+    name:                   `channel_${incr.next().value}`,
+    title:                  casual.title,
+    channel_type:           privateChannel ? "private" : "public",
+    description:            casual.description,
+    public_description:     casual.description,
+    num_users:              casual.integer(0, 500),
+    user_is_contributor:    casual.coin_flip,
+    user_is_subscriber:     casual.coin_flip,
+    user_is_moderator:      casual.coin_flip,
+    membership_is_managed:  casual.coin_flip,
+    allow_widgets_ui:       casual.coin_flip,
+    allow_related_posts_ui: casual.coin_flip,
```

Comment:
is this supposed to be added to the settings object? the `allow_widgets_ui` on the line above looks out of place as well.

Reply:
It is in the settings object [here](https://github.com/mitodl/open-discussions/pull/1797/files#diff-2792129aef83d6b2961cc36842c2df4eR24). Am I missing a different settings object where this needs to be? Also I'm not sure what you mean by `allow_widgets_ui` being out of place

Reply:
I was just wondering why it's been added to the channel factory here

Reply:
given that it isn't included in the API responses from the channel API

Reply:
Just saw your latest comment. 100% correct, `allow_widgets_ui ` shouldn't have been there in the first place. I removed it and `allow_related_posts_ui`

Reply:
haha, probably could have been more clear with my initial comment on that one :D